src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx

```diff
@@ -73,19 +88,20 @@ export const AssetSelectorScreen = ({assetTokens, onSelect, onSelectAll}: Props)
 }
 
 type AssetSelectorItemProps = {
-  assetToken: TokenEntry
-  onPress: (tokenEntry: TokenEntry) => void
+  wallet: YoroiWallet
+  tokenId: TokenId
+  amount: BigNumber
```

Comment:
string, convert to BigNumber only when is required.

Reply:
Done. Thanks